docs/resources/aws_guardduty_detector.md

```diff
@@ -0,0 +1,76 @@
+---
+title: About the aws_guardduty_detector Resource
+platform: aws
+---
+
+# Requirements
+
+Train AWS version 0.1.18 or newer is required for this resource.
+
+# aws\_ami
+
+Use the `aws_guardduty_detector` InSpec audit resource to test properties of a single AWS GuardDuty Detector.
+
+## Syntax
+
+An `aws_guardduty_detector` resource block declares the tests for a single AWS AMI by image id.
```

Comment:
should this say `detector id` rather than `image id`?

Reply:
Updated in latest commit.